app/epa/settings.py

```diff
@@ -171,7 +171,7 @@
 
 CRISPY_TEMPLATE_PACK = "bootstrap4"
 
-EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"

```

Comment:
We should use console only when debug is set to True (the credentials I used for stmp are obviously not stored in the github repo)

Reply:
Yes, your are right, I will set this PR to draft, fix this and then remove draft mode again and order an re-review.